src/Shipment/TaxIdentificationNumbers/Enums/TaxNumberTypeEnum.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MyParcelCom\Integration\Shipment\TaxIdentificationNumbers\Enums;
```

Comment:
There are no more enums for tax identification numbers, I'd suggest we move the class up in the TaxIdentificationNumbers namespace (as we do with Customs).
```php
namespace MyParcelCom\Integration\Shipment\TaxIdentificationNumbers;
```

Reply:
@zwaans ^^

Reply:
It seems I had forgotten to push this change. I do think this makes it harder to instantly see what is going on in the project structure (the removal of the folder I mean) I do agree on not suffixing the name with 'enum' on enums.